docs/modules/ROOT/pages/cops_layout.adoc

```diff
@@ -833,7 +864,7 @@ end
 | Name | Default value | Configurable values
 
 | Categories
-| `{"module_inclusion" => ["include", "prepend", "extend"]}`
+| `{"module_inclusion"=>["include", "prepend", "extend"]}`
```

Comment:
This document is automatically updated at release time, so no manual update is necessary.